spec/features/filling_in_a_police_per_spec.rb

```diff
@@ -13,10 +13,11 @@
 
     expect(current_path).to eq root_path
 
-    healthcare_data = build(:healthcare, :with_medications)
-    risk_data = build(:risk, :with_high_csra)
-    detainee = build(:detainee)
     move_data = build(:move, from_establishment: banbury_police_station)
+    escort = build(:escort, move: move_data)
```

Comment:
I wonder if we should consider at some staging using the rspec-examples gem to make these feature specs more readable? (https://github.com/railsware/rspec-example_steps). I do like the Given, When, Then syntax. We have a lot of boilerplate setup which could be in a Given block for example.